tests/python/contrib/test_cmsisnn/test_conv2d.py

```diff
@@ -124,7 +124,113 @@ def make_model(
 
 
 @tvm.testing.requires_cmsisnn
-@pytest.mark.parametrize("ifm_shape", [(1, 25, 25, 12), (1, 64, 100, 4)])
+@pytest.mark.parametrize("ifm_shape", [(1, 64, 100, 4)])
+@pytest.mark.parametrize("kernel_size", [(3, 3)])
+@pytest.mark.parametrize("padding", ["SAME", "VALID"])
+@pytest.mark.parametrize("strides, dilation", [((1, 1), (1, 1))])
+@pytest.mark.parametrize("relu_type", ["RELU"])
```

Comment:
Make these variables instead of parametrize so it's clear they don't change.

Reply:
Made the change. Let's see how the checks go.